kuromoji-core/src/main/java/com/atilika/kuromoji/fst/State.java

```diff
@@ -10,6 +26,8 @@
     boolean visited; //for visualization purpose
     private int targetJumpAddress = -1;
```

Comment:
This is used only for the old FSTCompiler so perhaps you can remove this to (and refactor it).